lib/main_common.pm

```diff
@@ -667,7 +667,7 @@ sub is_smt {
 }
 
 sub is_rmt {
-    return ((get_var("PATTERNS", '') || get_var('HDD_1', '')) =~ /rmt/) && is_sle('>=15');
+    return (get_var('RMT_TEST') && is_sle('>=15'));
```

Comment:
Please use check_var instead, as `get_var('RMT_TEST')` can return `undef` if it's not set